src/oaklib/implementations/simpleobo/simple_obo_implementation.py

```diff
@@ -813,8 +813,12 @@ def apply_patch(
             t = self._stanza(patch.about_node, strict=True)
             for tv in t.tag_values:
                 if tv.tag == TAG_DEFINITION:
-                    # The strip() portion is to handle a KGCL bug
-                    tv.replace_quoted_part(patch.new_value.strip("'"))
+                    if isinstance(patch, kgcl.RemoveTextDefinition):
```

Comment:
this should be at the same level as the other switches

Reply:
Done